pkg/driver/controller_test.go

```diff
@@ -0,0 +1,43 @@
+/*
+Copyright 2020 The OpenEBS Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package driver
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestRoundOff(t *testing.T) {
+
+	tests := map[string]struct {
+		input    int64
+		expected int64
+	}{
+		"Minimum allocatable is 1Mi": {input: 1, expected: Mi},
+		"roundOff to same Mi size":   {input: Mi, expected: Mi},
+		"roundOff to nearest Mi":     {input: Mi + 1, expected: Mi * 2},
+		"roundOff to same Gi size":   {input: Gi, expected: Gi},
+		"roundOff to nearest Gi":     {input: Gi + 1, expected: Gi * 2},
```

Comment:
Can we add tests for G and M  also? What is the expected result for (G) "1000 * 1000 * 1000"?

Reply:
1G will be 1000341504 (i.e. 954Mi).